frontend/src/pages/login/Login.tsx

```diff
@@ -1,25 +1,13 @@
-import { Dispatch, useState } from 'react';
+import { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { TextField, Button } from '@material-ui/core';
-import './Login.scss';
-
-interface Props {
-  setLoggedIn: Dispatch<boolean | undefined>;
-}
 
-interface ErrorResponse {
-  status: false;
-  error: string;
-}
-
-interface SuccessResponse {
-  status: true;
-  name: string;
-  token: string;
-}
+import { useAuth } from '../../contexts/Auth';
+import './Login.scss';
 
-export default function LoginV2({ setLoggedIn }: Props) {
+export default function LoginV2() {
```

Comment:
Not sure if the v2 was intended?

Reply:
Woopsies. I meant to remove that and forgot.